.node-version

```diff
@@ -0,0 +1 @@
+16
```

Comment:
I feel this should be the latest LTS (18)

Reply:
I used the one configured for deploy, should we change both? I don't know if there is any breaking thing though